.changeset/perfect-suits-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added message arguments to `declaration-property-unit-allowed-list`
```

Comment:
```suggestion
Added: message arguments to `declaration-property-unit-allowed-list`
```

Reply:
Agh! I think I've made a few of these issues. Let me resolve the same issue in #6558. Thank you for your quick review!